### ISSUE TITLE
'//': not a ZFS Filesystem

### ISSUE THREAD

**Issue by slynn1324** [zfsonlinux/zfs-auto-snapshot#86]
Installed on Ubuntu 16.04 from the Git repo.  The following command:
```
sudo zfs-auto-snapshot --label=frequent --keep=4 '//'
```
errors with:
```
'//': not a ZFS filesystem
Error: zfs list 1: 
```

The happens when running directly as above, and also causes the default cron.d jobs to fail as they use the same syntax.

**Comment by highvoltage** [comment]
Reverted PR85, mind pulling and trying again?

**Comment by EdvinOlofsson** [comment]
I'm having the same error. Any update on this? 
My pool is located at /mypool
Also, I have sub pools e.g /mypool/storage1 /mypool/storage2
How would I setup individual auto snapshots for each storage?

**Comment by milojah** [comment]
I am having the same issue on Centos 7.4.

Thanks.

**Comment by lycovian** [comment]
Same issue here.  Ubuntu 16.04.3 installed via:
```
curl -L https://github.com/zfsonlinux/zfs-auto-snapshot/archive/master.tar.gz | tar xz -C /tmp/
```

**Comment by jdh13** [comment]
Hello,
in my case, i see the same errors because root is not zfs. 
In files cron.d/zfs-auto-snapshot   and cron.hourly/zfs-auto-snapshot (and perhaps others if you use them), you can replace '//' by the list of zpool names and it works
hth

**Comment by seonwoolee** [comment]
I have a pool named master. I have a bunch of datasets under master. Lets say they are
```
master/Alice
master/Alice/home
master/Bob
master/Bob/home
master/Eve
```
I've set `com.sun:auto-snapshot=true` for master/Alice, master/Alice/home, and master/Bob. I've set it `false` for master, master/Bob/home, and master/Eve.
 
If I run `sudo zfs-auto-snapshot --label=frequent --keep=4 //` I get the error like the others above.
If I run `sudo zfs-auto-snapshot --label=frequent --keep=4 master` no snapshots get created.
If I run `sudo zfs-auto-snapshot --label=frequent --keep=4 master/Alice` snapshots get created for `master/Alice` and `master/Alice/home`. 
If I run sudo zfs-auto-snapshot --label=frequent --keep=4 master/Bob` no snapshots get created.
And no snapshots are created for `master/Eve`.

So based on this it seems I need to list every dataset I want to have snapshotted? And if a parent has `com.sun:auto-snapshot=false`, calling `zfs-auto-snapshot` on the parent will not create any snapshots for it's children?

But I thought the entire point of having `com.sun:auto-snapshot=true` for each dataset was that I could run the `zfs-auto-snapshot` and have it automatically snapshot the ones for which `com.sun:auto-snapshot` is set to `true`?
